src/pages/authors/index.astro

```diff
@@ -1,90 +1,5 @@
 ---
-import Layout from '@/components/layout.astro'
-import Panel from '@/components/panel.astro'
-import { Image } from 'astro:assets'
-import { twMerge } from 'tailwind-merge'
-import lazar from '@/assets/discord-avatars/member-1.webp'
-import tentacle from '@/assets/authors/tentacle.png'
-import WriteBanner from '@/components/write-banner.astro'
-import PostCard from '@/components/post-card.astro'
-import { getCollection } from 'astro:content'
+export const prerender = false
 
-const { class: className = '' } = Astro.props
-
-const author = {
-  name: 'Lazar Nikolov',
-  handle: 'nikolovlazar',
-  badges: ['content creator', 'devrel', 'software engineer'],
-}
-
-const posts = await getCollection('blog')
+return Astro.redirect('/authors/lazar-nikolov')
```

Comment:
Let's get rid of this whole page and just add a link to the author in the `PostCard` component.